libs/designsystem/src/lib/components/chart/options/bar.ts

```diff
@@ -0,0 +1,107 @@
+import { Options } from 'highcharts';
+
+import { DesignTokenHelper } from '../../../helpers/design-token-helper';
+
+const getColor = DesignTokenHelper.getColor;
+const fontSize = DesignTokenHelper.fontSize;
+
+function colorPoints() {
+  var series = this.series;
+  for (var i = 0, ie = series.length; i < ie; ++i) {
+    var points = series[i].data;
+    for (var j = 0, je = points.length; j < je; ++j) {
+      if (points[j].graphic) {
+        points[j].graphic.element.style.stroke = getColor('background-color').value;
+      }
+    }
+  }
+}
+
+export const getBarOptions = (data: number[], categories: string[]) =>
```

Comment:
I think this is better than injecting as we can pass params

Reply:
Let's have a talk about this, I don't see why this should differ from the rest...